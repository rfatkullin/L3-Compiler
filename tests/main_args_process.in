func main (args->[[char]]) -> int
	argsCnt = length(args) -> int;

	print argsCnt;

	for i = 0 -> int to argsCnt - 1 do
		print args[i];
	endfor;

	main = 0;
endfunc
