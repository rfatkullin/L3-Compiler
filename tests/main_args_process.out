5
first
second
third
4
5
